roles/openshift_storage_glusterfs/tasks/uninstall.yml

```diff
@@ -0,0 +1,10 @@
+---
+- block:
+  - include_tasks: glusterfs_config_facts.yml
+  - include_tasks: glusterfs_uninstall.yml
+    when: groups.glusterfs | length > 0
```

Comment:
The group may not always be defined, so check that with `'glusterfs_registry' in groups and groups['glusterfs_registry'] | length > 0`.

Reply:
I have updated it - please check